kotsadm/pkg/snapshot/backup.go

```diff
@@ -26,32 +26,36 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 )
 
-func CreateBackup(a *apptypes.App) error {
-	if err := createApplicationBackup(context.TODO(), a); err != nil {
-		return errors.Wrap(err, "failed to create application backup")
+func CreateBackup(a *apptypes.App, isScheduled bool) (*velerov1.Backup, error) {
+	backup, err := createApplicationBackup(context.TODO(), a, isScheduled)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to create application backup")
 	}
 
 	// uncomment to create disaster recovery snapshots
-	// if err := createAdminConsoleBackup(context.TODO()); err != nil {
-	// 	return errors.Wrap(err, "failed to create admin console backup")
+	// if err := createAdminConsoleBackup(context.TODO(), isScheduled); err != nil {
+	// 	return nil, errors.Wrap(err, "failed to create admin console backup")
```

Comment:
i like how you fixed commented out code here :)

Reply:
🙂